mne/viz/tests/test_topomap.py

```diff
@@ -34,6 +33,7 @@
 from mne.viz.topomap import (_get_pos_outlines, _onselect, plot_topomap,
                              plot_arrowmap, plot_psds_topomap)
 from mne.viz.utils import _find_peaks, _fake_click
+from mne.viz.tests.test_topo import _gen_nirs_data
```

Comment:
The modern way to do this would be to create a `mne/viz/tests/conftest.py` and add a:
```
@pytest.fixture()
def fnirs_evoked():
    """Create a fnirs evoked."""
    ...
    return evoked
```